apps/system/test/marionette/lib/rocketbar.js

```diff
@@ -26,9 +26,7 @@ Rocketbar.clientOptions = {
   },
   settings: {
     'homescreen.manifestURL':
-      'app://verticalhome.gaiamobile.org/manifest.webapp',
-    'ftu.manifestURL': null,
-    'lockscreen.enabled': false
+      'app://verticalhome.gaiamobile.org/manifest.webapp'
```

Comment:
if you remove the default homescreen manifestURL earlier, you can remove it here too ?